fluentbit_config/parsers.conf

```diff
@@ -0,0 +1,6 @@
+# parsers.conf
+[PARSER]
+    Name        json
+    Format      json
+    Time_Key    time
+    Time_Format %Y-%m-%dT%H:%M:%SZ
```

Comment:
Is the `Z` suffix intentional, i.e. for UTC?

Reply:
the parsers.conf was not used anymore, so removed